pkgs/ffigen/tool/libclang_config.yaml

```diff
@@ -43,6 +43,9 @@ enums:
     - CXObjCPropertyAttrKind
     - CXTypeNullabilityKind
     - CXTypeLayoutError
+  as-int:
+    include:
+      - .*
```

Comment:
Off-topic: Does the default cause issues for libclang? If yes, can/should we address that? Should this be in a separate PR?

Reply:
Some of the enums are flags. For example, we need to `|` together the values of `CXTranslationUnit_Flags` in `lib/src/header_parser/parser.dart:102`. This is the first time the bindings have had to be regenerated since the enum changes.